src/sass/switch/_base.scss

```diff
@@ -0,0 +1,128 @@
+// Copyright (C) 2023 The Trustees of Indiana University
+// SPDX-License-Identifier: BSD-3-Clause
+
+@use '../core' as *;
+
+.#{$prefix}-switch {
+  align-items: center;
+  background: none;
+  background-color: $color-black-400;
+  border: $spacing-xxs/2 solid $color-black-400;
+  border-radius: $border-radius-circle;
+  color: $color-black-100;
+  display: flex;
+  height: $spacing-xl + $spacing-xxs;
+  line-height: 1.1;
+  padding: $spacing-xxs;
+  position: relative;
+  -webkit-appearance: none;
+
+  &::after {
+    background-color: $color-black-000;
+    border-radius: $border-radius-circle;
+    background-image: url("data:image/svg+xml,%3Csvg width='16' height='16' viewBox='0 0 16 16' xmlns='http://www.w3.org/2000/svg' version='1.1'%3E%3Cpath fill-rule='evenodd' clip-rule='evenodd' d='M3.41421 2L7.91421 6.5L12.4142 2L13.8284 3.41421L9.32843 7.91421L13.8284 12.4142L12.4142 13.8284L7.91421 9.32843L3.41421 13.8284L2 12.4142L6.5 7.91421L2 3.41421L3.41421 2Z' fill='%234C5A69'/%3E%3C/svg%3E");
+    background-size: 16px 16px;
```

Comment:
Let's use design tokens here instead of hard-coded pixel values—should be `$spacing-sm`